### DISCUSSION TITLE
Error Message at Viewing Metrics

### DISCUSSION THREAD

**Discussion by kazukimisawa** [DaxStudio/DaxStudio#927] [Q&A] [answered]
Hello, I have upgraded my DAX Studio to version 3.
After that, I got the following error at viewing metrics, and I cannot see it... Can I get help, please?

Kazuki

![image](https://user-images.githubusercontent.com/113003782/188791898-5fd0e5c9-ca56-4671-b635-c16aaee214a2.png)

**Comment by dgosbell** [comment, marked as answer]
This appears to only affect certain data models. We already have this logged under issue #887 and there will be a fix for this included in the next release.

**Reply by kazukimisawa** [reply]
Thank you so much, and noted!
